x-pack/plugins/advanced_ui_actions/public/plugin.ts

```diff
@@ -88,11 +97,32 @@ export class AdvancedUiActionsPublicPlugin
     });
     uiActions.addTriggerAction(PANEL_BADGE_TRIGGER, timeRangeBadge);
 
+    const isUrlDrilldownAvailable = (licence: ILicense) =>
+      licence.hasAtLeast('gold') && licence.isActive;
+    this.registerUrlDrilldownSubscription = licensing.license$
```

Comment:
Is this register / unregister correct approach? 🤔 
Or should it be handled inside isCompatible? 

Maybe uiActions framework should simplify this allowing to put `gold` on action definition?